### PR TITLE
docs(kafka introduction): editorial review

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

A few suggested changes from an editorial review

### CODE REVIEW

**Comment by PaulRMellor** [comment]
Hi @fvaleri -- I read through the first scenario and made a few suggestions.

File: sessions/001/README.md [hunk 1]
```diff
@@ -4,59 +4,59 @@
...
-One of the brokers is elected as controller, which has the additional responsibility of managing the states of partitions and replicas and for performing administrative tasks like reassigning partitions.
-In ZooKeeper mode there is only one elected controller, while in the new KRaft mode we have a quorum of controllers, where only one is active at any time, while the others are ready to take over in case of failure.
+There are also two higher level APIs: the connect API for integration with external systems and the streams API for stream processing.
+Within a cluster, the control plane handles cluster metadata and the data plane handles user data.
+A broker in a cluster is elected as controller, which has the additional responsibility of managing the states of data partitions and replicas and for performing administrative tasks like reassigning partitions.
+In ZooKeeper mode there is only one elected controller, while in the new KRaft mode we have a quorum of controllers. One broker is active at any time, while the others are ready to take over in case of failure.
```
Comment (fvaleri) [inline sessions/001/README.md:13]: Maybe this should be in two separate lines (one sentence per row).

File: sessions/001/README.md [hunk 2]
```diff
@@ -4,59 +4,59 @@
...
+Each message is modeled as a record with timestamp, key, value, and optional headers.
+Key and value are just byte arrays, which gives you the flexibility to encode the data in whatever format you want using your favorite serializer.
 The timestamp is always present, but it can be set by the application when it sends (default), or by the Kafka runtime when it receives.
 Whenever possible, records are buffered and sent in batches (a single request can include multiple batches, one for each partition).
 
 Records are stored in a topic, which is further divided into one or more partitions, distributed evenly across the brokers.
 Each partition is stored on disk as a series of fixed-size commit logs called segments.
-Ordering is only guaranteed at the partition level.Each record within a partition has a unique id called the offset, which is a monotonically increasing number that is never reused.
+Ordering is only guaranteed at the partition level. Each record within a partition has a unique id called the offset, which is a monotonically increasing number that is never reused.
```
Comment (fvaleri) [inline sessions/001/README.md]: Maybe this should be in two separate lines (one sentence per row).

File: sessions/001/README.md [hunk 3]
```diff
@@ -4,59 +4,59 @@
...
 Within a consumer group, a partition is assigned to exactly one consumer to not break ordering, but that consumer can handle multiple partitions.
-Each consumer periodically or manually commits its position (next offsets to read) to an internal topic called `__consumer_offsets`.
+Each consumer periodically or manually commits its position (the next offsets to read) to an internal topic called `__consumer_offsets`.
 
-# Example: deploy on localhost
+# Example: deploy a Kafka cluster on localhost
 
-We are going to deploy a Kafka cluster on your local machine.This is useful for quick tests where a multi node cluster is not required.
-We will use the latest upstream release because the downstream release is just a rebuild with few additional and optional plugins.
+In this example, we deploy a Kafka cluster on a local machine. This is useful for quick tests where a multi node cluster is not required.
```
Comment (fvaleri) [inline sessions/001/README.md]: Maybe this should be in two separate lines (one sentence per row).

File: sessions/001/README.md [hunk 4]
```diff
@@ -126,7 +126,7 @@ total 12K
 -rw-r--r--. 1 fvaleri fvaleri  43 Sep  8 16:55 partition.metadata
 ```
 
-Partition log files are in binary format, but Kafka includes a dump tool for decoding them.On this partition we have one batch (`baseOffset`), containing only one record (`| offset`) with key "1" and value "hello".
+Partition log files are in binary format, but Kafka includes a dump tool for decoding them. On this partition, we have one batch (`baseOffset`), containing only one record (`| offset`) with key "1" and value "hello".
```
Comment (fvaleri) [inline sessions/001/README.md]: Maybe this should be in two separate lines (one sentence per row).

**Review by fvaleri — COMMENTED** [review]
Hi @PaulRMellor, thanks.

Looks good, just add some carriage returns and I'll merge :)

**Pull request merged**